apps/freelance/src/redux/services/authApi.ts

```diff
@@ -0,0 +1,21 @@
+import { BACK_HOST, BACK_PORT, BACK_PROTO } from '@freelance/components';
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const authApi = createApi({
+  reducerPath: 'authReducer',
+  baseQuery: fetchBaseQuery({
+    baseUrl: `${BACK_PROTO}://${BACK_HOST}:${BACK_PORT}/auth`,
```

Comment:
its too complicated, when you will push to server, you will not have port (not really like that but you will not see it), it better use one variable

Reply:
fixed